worlds/rabi_ribi/items.py

```diff
@@ -1,41 +1,139 @@
 """This module represents item definitions for Rabi-Ribi"""
-from typing import List
+from typing import Dict, List, NamedTuple, Optional, Set
 
-from BaseClasses import Item
+from BaseClasses import Item, ItemClassification
 from .existing_randomizer.visualizer import load_item_locs
 
+rabi_ribi_base_id: int = 8350438193300
```

Comment:
Might make more sense to stick this ID in `utility.py` (with a `get_rabi_ribi_base_id()`). This ID is used for locations as well as items.